circuit/types/group/src/helpers/from_x_coordinate.rs

```diff
@@ -28,8 +28,8 @@ impl<E: Environment> Group<E> {
     }
 
     /// Initializes an affine group element from a given x-coordinate field element.
-    /// Also returns an error flag, set if there is no group element with the given x-coordinate;
-    /// in that case, the returned point is `(0, 0)`, but immaterial.
+    /// Returns an error flag, indicating if there is a group element with the given x-coordinate;
```

Comment:
I would read 'indicating' as 'when set', in which case it should say 'there is no group element'. I think that if we use the 'set' terminology explicitly it's more clear.